explore_the_world/data/build.sh

```diff
@@ -13,11 +13,14 @@ set -o pipefail
 
 # STEP 0: initialize config variables
 : "${DATA_IMAGE_NAME:=deepset/elasticsearch-countries-and-capitals}"
+: "${HAYSTACK_IMAGE_NAME:=deepset/haystack:cpu-main}"
+: "${DATASET_DIR:=dataset}"
+: "${NETWORK:=explore_the_world}"
 
 if [ -z "$DATA_IMAGE_PLATFORM" ]
 then
     image_name=$DATA_IMAGE_NAME
-    build_cmd="docker build -t $DATA_IMAGE_NAME -f Dockerfile.elasticsearch ."
+    build_cmd="docker build --push -t $DATA_IMAGE_NAME -f Dockerfile.elasticsearch ."
```

Comment:
Why pushing here? At this point this is just a regular ES image containing no data, the goal of the build script is pushing at step 8 after data is indexed.

Reply:
Good catch, I put that in while testing the script. Now removed.